src/test/java/com/passbook/sparkeighteen/service/TransactionServiceTest.java

```diff
@@ -101,6 +101,7 @@ public void updateTransaction_transactionIdNotExists_transactionUpdateUnSuccessf
                 .transactionType(TransactionType.DEBIT)
                 .build();
 
+
```

Comment:
remove extra line

Reply:
line 103 is extra
How are you guys not seeing it makes doubt do you really read the review?